subsys/bluetooth/audio/shell/cap_commander.c

```diff
@@ -662,6 +673,87 @@ static int cmd_cap_commander_broadcast_reception_start(const struct shell *sh, s
 
 	return 0;
 }
+
+static int cmd_cap_commander_broadcast_reception_stop(const struct shell *sh, size_t argc,
+						      char *argv[])
+{
+	struct bt_cap_commander_broadcast_reception_stop_member_param
+		member_params[CONFIG_BT_MAX_CONN] = {0};
+	struct bt_cap_commander_broadcast_reception_stop_param param = {
+		.type = BT_CAP_SET_TYPE_AD_HOC,
+		.param = member_params,
+	};
+	struct bt_cap_commander_broadcast_reception_stop_member_param *member_param =
+		&member_params[0];
+
+	struct bt_conn *connected_conns[CONFIG_BT_MAX_CONN] = {0};
+	size_t conn_cnt = 0U;
+	unsigned long src_id;
+
+	int err = 0;
+
+	if (default_conn == NULL) {
+		shell_error(sh, "Not connected");
+		return -ENOEXEC;
+	}
+
+	/* TODO: Add support for coordinated sets */
+
+	/* Populate the array of connected connections */
+	bt_conn_foreach(BT_CONN_TYPE_LE, populate_connected_conns, (void *)connected_conns);
+	for (size_t i = 0; i < ARRAY_SIZE(connected_conns); i++) {
+		struct bt_conn *conn = connected_conns[i];
+
+		if (conn == NULL) {
+			break;
+		}
+
+		conn_cnt++;
+	}
+
+	src_id = shell_strtoul(argv[2], 0, &err);
+	if (err != 0) {
+		shell_error(sh, "Could not parse src_id: %d", err);
+
+		return -ENOEXEC;
+	}
+
+	if (src_id > UINT8_MAX) {
+		shell_error(sh, "Invalid src_id: %lu", src_id);
+
+		return -ENOEXEC;
+	}
+
+	member_param->src_id = src_id;
+	/* TODO: Allow for multiple subgroups */
+	member_param->num_subgroups = 1;
+	member_param->member.member = connected_conns[0];
+
+	/* each connection has its own member_params field
+	 * here we use the same values for all connections, so we copy
+	 * the parameters
+	 */
```

Comment:
The `src_id` may not be the same for each connection, so we cannot use this approach here. 

We need a `src_id` _per_ connection